File: Fitness.Console/src/app/services/food.dto.ts
export class FoodDto {
	id: number;
	name: string;
	calories: number;
	carbs: number;
	fat: number;
	sodium: number;
	fiber: number;

	constructor() {
		this.id = 0;
		this.name = '';
		this.calories = 0;
		this.carbs = 0;
		this.fat = 0;
		this.sodium = 0;
		this.fiber = 0;
	}
}

File: Fitness.Api.Docker/Dtos/FoodDto.cs
// ***************************************************************
// Copyright 2020 MorrData LLC. All rights reserved.
// ***************************************************************

namespace Fitness.Api.Dtos
{
    public class FoodDto
    {
        public long FoodId { get; set; }
        public string Name { get; set; }
        public int Calories { get; set; }
        public double ServiingSize { get; set; }
        public int Unit { get; set; }
        public int Carbs { get; set; }
        public int Fat { get; set; }
        public int Salt { get; set; }
        public int Protein { get; set; }
        public int Sugar { get; set; }
    }
}
